src/NLopt.jl

```diff
@@ -633,9 +630,11 @@ end
 optimize(o::Opt, x::AbstractVector{<:Real}) =
   optimize!(o, copyto!(Array{Cdouble}(undef,length(x)), x))
 
-############################################################################
-
-include("MPB_wrapper.jl")
-include("MOI_wrapper.jl")
+if !isdefined(Base, :get_extension)
+    include("../ext/MathOptInterfaceExtension.jl")
+    # so that the struct defined there is added to this module in that module's init
```

Comment:
Is this comment accurate? This branch only runs pre-1.9, and `MathOptInterfaceExtension.__init__` only calls `setglobal!` for >= 1.9.

Reply:
No, not anymore. I deleted it.